src/EnergyPlus/OutputProcessor.cc

```diff
@@ -894,28 +894,17 @@ namespace OutputProcessor {
 
         // METHODOLOGY EMPLOYED:
         // Look it up in a list of valid index types.
-
-        // FUNCTION LOCAL VARIABLE DECLARATIONS:
-
-        // TODO: , "HEATBALANCE", "HEAT BALANCE" are used nowhere aside from tests. Should we remove them?
-        std::vector<std::string> zoneIndexes({"ZONE", "HEATBALANCE", "HEAT BALANCE"});
-        std::vector<std::string> systemIndexes({"HVAC", "SYSTEM", "PLANT"});
-        std::string uppercase(UtilityRoutines::MakeUPPERCase(TimeStepTypeKey));
```

Comment:
Got rid of these vectors and loops and just replaced it with a switch block now that the input parameter is an enum and not a string.